.mailmap

```diff
@@ -1017,6 +1017,7 @@ Peter Enenkel <peter.enenkel+git@gmail.com> <peter.enenkel+github@gmail.com>
 Peter Schmidt <peter@peterjs.com>
 Peter Stangl <peter.stangl@ph.tum.de>
 Petr Kungurtsev <corwinat@gmail.com> Corwinpro <corwinat@gmail.com>
+Phil LeMaitre <112662371+AntiPhoton47@users.noreply.github.com>
```

Comment:
You can set this to your personal email like this:
```
Phil LeMaitre <personal@email.com> Phil LeMaitre <112662371+AntiPhoton47@users.noreply.github.com>
```
Then the first name and email address will be recorded in the AUTHORS file but the .mailmap will record that any commits using the second name and email address are from the same person.

Reply:
Ah, thank you for letting me know! I just changed the .mailmap file to reflect this.

Reply:
Semantics for such changes are in the `.mailmap` file docstring at the top of the file. In this case I believe the repetition of your name is not necessary, just 

`name <standard address> <alt address>`